### ISSUE TITLE
Support for i18n

### ISSUE THREAD

**Issue by tdlage** [auth0/react-native-lock#51]
I need the login screen in different languages, is it possible right now ?

**Comment by camel113** [comment]
+1

**Comment by KBLNY** [comment]
+1

**Comment by KBLNY** [comment]
Obviously, this is possible by passing a `dict` parameter to your `lock.show()` method. However, this parameter is not handle in Native code, hence it won't work. (cf. [https://auth0.com/docs/libraries/lock/customization#dict-string-object-](https://auth0.com/docs/libraries/lock/customization#dict-string-object-) for references).

You can, for now, add a regular localization file (for iOS, I didn't test yet for Android) nammed `Lock.strings` and put your translations here, like (here is a sample for French) :

```
/* 
  Lock.strings
  Sidemash

  Created by Kevin on 07/06/16.
  Copyright © 2016 Facebook. All rights reserved.
*/

/* Change Password */

"Please enter your email address. We will send you an email to reset your password."="Merci de renseigner votre email. Nous vous enverons un email pour réinitialiser votre mot de passe.";
"SEND"="Envoyer";
"Send reset password email"="Réinitialiser le mot de passe";
"Reset Password"="Réinitialiser le mot de passe";
"We've just sent you an email to reset your password."="Un email vous a été envoyé pour réinitialiser votre mot de passe.";
"Couldn't request to reset your password"="Impossible de traiter votre demande";


/* Credential Fields */

"Email"="Email";
"Email/Username"="Email/nom d'utilisateur";
"Password"="Mot de passe";
"Username"="Nom d'utilisateur";
"Phone Number"="Numéro de téléphone";
"Verification Code"="Code de vérification";

// ...
```

I will come with a full `Lock.strings` file later if no better solution

**Comment by ivanzotov** [comment]
@KBLNY thank you so much, I just put Lock.strings file with translations into root of ios project and it works

**Comment by mariomelo** [comment]
You can do the same thing in Android by creating the corresponding entries on res/values/strings.xml.

The keys can be found in the exploded aar in a file called values.xml. Just do a file search for "Reset your password" inside your project (after you've built it at least once) and you'll find it.

**Comment by pbassut** [comment]
Will this ever be supported? Sounds like a very simple thing.
I couldn't get it working using a Lock.strings on the root project.
Maybe it doesn't work with newer versions of react-native?

**Comment by pbassut** [comment]
No, wait. It did work. I just couldn't notice because the example doesn't change very noticeable things. 
Sorry.

Thanks @KBLNY

**Comment by thiagoribeir015** [comment]
You can also follow this tutorial to make it work for multiple languages on iOS:
https://auth0.com/docs/libraries/lock-ios/v2/internationalization
